tests/integration/api_v3/http/blockchainAppsMetaList.test.js

```diff
@@ -75,13 +75,13 @@ describe('Blockchain application meta list API', () => {
 		const response = await api.get(`${endpoint}?chainName=Lisk`);
 		expect(response).toMap(goodRequestSchema);
 		expect(response.data).toBeInstanceOf(Array);
-		expect(response.data.length).toEqual(1);
+		expect(response.data.length).toBeGreaterThanOrEqual(1);
```

Comment:
Search with `chainName` should always return either zero or one result. It must be an exact match. For pattern search we have the `search` param. Please fix the test accordingly.

Reply:
Clarified offline!